UnityProject/Assets/Scripts/Managers/AdminCommandsManager.cs

```diff
@@ -54,14 +54,13 @@ private void Awake()
 		/// <param name="adminToken"></param>
 		/// <param name="sender">The client which sends the command, this is populated by mirror so doesnt need to be manually
 		/// put in the parameters when calling the commands</param>
-		public static bool IsAdmin(string adminId, string adminToken, NetworkConnectionToClient sender)
+		public static bool IsAdmin(NetworkConnectionToClient sender, out ConnectedPlayer player)
```

Comment:
```suggestion
		public static bool IsAdmin(NetworkConnection sender, out ConnectedPlayer player)
```

Should fix the tests